orchestrator/src/server.ts

```diff
@@ -355,6 +424,17 @@ app.get("/authenticate/:uuid/:check", async (req, res) => {
     }
 });
 
+async function getResponseFromCache(hashedKey: string){
```

Comment:
Are these functions just meant to provide an abstraction around caching server calls?